src/core/net/ip6.hpp

```diff
@@ -106,6 +107,10 @@ class Ip6 : public InstanceLocator
     {
         kDefaultHopLimit   = OPENTHREAD_CONFIG_IPV6_DEFAULT_HOP_LIMIT,
         kMaxDatagramLength = OPENTHREAD_CONFIG_IPV6_DEFAULT_MAX_DATAGRAM,
+#if OPENTHREAD_ENABLE_IP6_FRAGMENTATION
+        kMaxAssembledDatagramLength = OPENTHREAD_CONFIG_IPV6_MAX_ASSEMBLED_DATAGRAM,
+        kIp6ReassemblyTimeoutMilli  = OPENTHREAD_CONFIG_IP6_REASSEMBLY_TIMEOUT,
+#endif // OPENTHREAD_ENABLE_IP6_FRAGMENTATION
```

Comment:
nit: can avoid the `// OPENTHREAD_ENABLE_IP6_FRAGMENTATION` for small blocks (e.g. blocks of less than 20 lines of code).

Reply:
removed